exercises/practice/largest-series-product/LargestSeriesProduct.php

```diff
@@ -26,6 +26,11 @@
 
 class Series
 {
+    public function __construct(string $input)
+	{
```

Comment:
Replace tab by 4 spaces:
```suggestion
    {
```